cypress/support/custom_commands/login.js

```diff
@@ -2,80 +2,83 @@ const emailInputFieldElement = '[data-testid="username-email"]'
     const env = Cypress.env()
-    let link
-    let environmentUpperCased = environment.toUpperCase()
-    if (environmentUpperCased === 'BRB') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
-    } else if (environmentUpperCased === 'NBC') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
+    const environmentUpperCased = environment.toUpperCase()
+    const link = Cypress.config('baseUrl', env[environmentUpperCased])
+    cy.visit(link)
+    if (environmentUpperCased === 'NBC') {
       cy.get(nbcLoginWithEmailOptionButton).eq(1).click()
-    } else if (environmentUpperCased === 'DEFAULT') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
+    }
+    if (environmentUpperCased === 'DEFAULT') {
+      cy.visit('/login');
     }
 
     let userEmail
     let userPassword
+    let doExternalLogin = false
 
     switch (username) {
-
       case 'teacher':
         userEmail = 'TEACHER_1_EMAIL'
         userPassword = 'TEACHER_1_PASSWORD'
         break;
-
       case 'teacher1':
         userEmail = 'TEACHER_1_EMAIL'
         userPassword = 'TEACHER_1_PASSWORD'
         break;
-
       case 'teacher2':
         userEmail = 'TEACHER_2_EMAIL'
         userPassword = 'TEACHER_2_PASSWORD'
         break;
-
       case 'student':
         userEmail = 'STUDENT_1_EMAIL'
         userPassword = 'STUDENT_1_PASSWORD'
         break;
-
       case 'student1':
         userEmail = 'STUDENT_1_EMAIL'
         userPassword = 'STUDENT_1_PASSWORD'
         break;
-
       case 'student2':
         userEmail = 'STUDENT_2_EMAIL'
         userPassword = 'STUDENT_2_PASSWORD'
         break;
-
       case 'admin':
         userEmail = 'ADMIN_1_EMAIL'
         userPassword = 'ADMIN_1_PASSWORD'
         break;
-
       case 'expert':
         userEmail = 'EXPERT_1_EMAIL'
         userPassword = 'EXPERT_1_PASSWORD'
         break;
+      case 'student_extern':
+        userEmail = 'STUDENT_EXTERN'
+        userPassword = 'STUDENT_EXTERN_PASSWORD'
+        doExternalLogin = true
+        break;
+    }
+    if (doExternalLogin){
+      cy.get(defaultLoginViaExternalBroker).click()
+      cy.get(externalUsernameInputFieldElement).type(env[userEmail], {log:false})
+      cy.get(externalPasswordInputFieldElement).type(env[userPassword], {log:false}).type('{enter}')
+    } else {
+      cy.get(emailInputFieldElement).type(env[userEmail], {log:false})
+      cy.get(passwordInputFieldElement).type(env[userPassword], {log:false})
+      cy.get(submitButton).click()
     }
-    cy.get(emailInputFieldElement).eq(1).type(env[userEmail], {log:false})
-    cy.get(passwordInputFieldElement).eq(1).type(env[userPassword], {log:false})
-    cy.get(submitButton).eq(1).click()
     cy.url().should('contain', '/dashboard')
     cy.get(initials).click()
     cy.get(languageSelection).click()
     cy.get(languageDe).click()
-
   })
   cy.visit('/dashboard')
```

Comment:
here also cy.visit('/') is fine and 'dashboard' can be removed, its calling/checking it in line 78.